bundles/org.openhab.binding.teleinfo/src/main/resources/OH-INF/thing/common-lsm-channel-group.xml

```diff
@@ -5,8 +5,8 @@
 	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
 
 	<channel-group-type id="commonLSMGroupType">
-		<label>CommonLSMChannelGroup</label>
-		<description>Linky Standard Mode common channels</description>
+		<label>Linky Standard Mode Common Channels</label>
```

Comment:
Labels should be as concise as possible. You could remove the redundencies and name them Common, Producer and Three-phase

Reply:
Done!